src/service.js

```diff
@@ -1,166 +1,160 @@
biAgICBvblJlbG9hZChvbGRTZXNzaW9uSWQsIG5ld1Nlc3Npb25JZCkge1xuICAgICAgICBpZiAoIXRoaXMuaXNTZXJ2aWNlRW5hYmxlZCkge1xuICAgICAgICAgICAgLyogaXN0YW5idWwgaWdub3JlIG5leHQgKi9cbiAgICAgICAgICAgIHJldHVyblxuICAgICAgICB9XG5cbiAgICAgICAgY29uc3Qgc3RhdHVzID0gJ3N0YXR1czogJyArICh0aGlzLmZhaWx1cmVzID4gMCA/ICdmYWlsZWQnIDogJ3Bhc3NlZCcpXG5cbiAgICAgICAgaWYgKCFnbG9iYWwuYnJvd3Nlci5pc011bHRpcmVtb3RlKSB7XG4gICAgICAgICAgICBsb2cuaW5mbyhcbiAgICAgICAgICAgICAgICBgVXBkYXRlIChyZWxvYWRlZCkgam9iIHdpdGggc2Vzc2lvbklkICR7b2xkU2Vzc2lvbklkfSwgJHtzdGF0dXN9YFxuICAgICAgICAgICAgKVxuICAgICAgICAgICAgcmV0dXJuIHRoaXMudXBkYXRlSm9iKG9sZFNlc3Npb25JZCwgdGhpcy5mYWlsdXJlcywgdHJ1ZSlcbiAgICAgICAgfVxuXG4gICAgICAgIGNvbnN0IGJyb3dzZXJOYW1lID0gZ2xvYmFsLmJyb3dzZXIuaW5zdGFuY2VzLmZpbHRlcihcbiAgICAgICAgICAgIGJyb3dzZXJOYW1lID0+XG4gICAgICAgICAgICAgICAgZ2xvYmFsLmJyb3dzZXJbYnJvd3Nlck5hbWVdLnNlc3Npb25JZCA9PT0gbmV3U2Vzc2lvbklkXG4gICAgICAgIClbMF1cbiAgICAgICAgbG9nLmluZm8oXG4gICAgICAgICAgICBgVXBkYXRlIChyZWxvYWRlZCkgbXVsdGlyZW1vdGUgam9iIGZvciBicm93c2VyICcke2Jyb3dzZXJOYW1lfScgYW5kIHNlc3Npb25JZCAke29sZFNlc3Npb25JZH0sICR7c3RhdHVzfWBcbiAgICAgICAgKVxuICAgICAgICByZXR1cm4gdGhpcy51cGRhdGVKb2Iob2xkU2Vzc2lvbklkLCB0aGlzLmZhaWx1cmVzLCB0cnVlLCBicm93c2VyTmFtZSlcbiAgICB9XG5cbiAgICAvKiBpc3RhbmJ1bCBpZ25vcmUgbmV4dCAqL1xuICAgIGFzeW5jIHVwZGF0ZUpvYihzZXNzaW9uSWQsIGZhaWx1cmVzLCBjYWxsZWRPblJlbG9hZCA9IGZhbHNlLCBicm93c2VyTmFtZSkge1xuICAgICAgICBjb25zdCBib2R5ID0gdGhpcy5nZXRCb2R5KGZhaWx1cmVzLCBjYWxsZWRPblJlbG9hZCwgYnJvd3Nlck5hbWUpXG4gICAgICAgIGF3YWl0IG5ldyBQcm9taXNlKChyZXNvbHZlLCByZWplY3QpID0+IHtcbiAgICAgICAgICAgIHRoaXMuYXBpLnVwZGF0ZVNlc3Npb25CeUlkKHNlc3Npb25JZCwgYm9keSwgKGVyciwgcmVzdWx0KSA9PiB7XG4gICAgICAgICAgICAgICAgaWYgKGVycikge1xuICAgICAgICAgICAgICAgICAgICByZXR1cm4gcmVqZWN0KGVycilcbiAgICAgICAgICAgICAgICB9XG4gICAgICAgICAgICAgICAgcmV0dXJuIHJlc29sdmUocmVzdWx0KVxuICAgICAgICAgICAgfSlcbiAgICAgICAgfSlcbiAgICAgICAgdGhpcy5mYWlsdXJlcyA9IDBcbiAgICB9XG5cbiAgICAvKiBpc3RhbmJ1bCBpZ25vcmUgbmV4dCAqL1xuICAgIGdldEJvZHkoZmFpbHVyZXMsIGNhbGxlZE9uUmVsb2FkID0gZmFsc2UsIGJyb3dzZXJOYW1lKSB7XG4gICAgICAgIGxldCBib2R5ID0ge31cbiAgICAgICAgaWYgKCEoIWdsb2JhbC5icm93c2VyLmlzTXVsdGlyZW1vdGUgJiYgdGhpcy5jYXBhYmlsaXRpZXMubmFtZSB8fCBnbG9iYWwuYnJvd3Nlci5pc011bHRpcmVtb3RlICYmIHRoaXMuY2FwYWJpbGl0aWVzW2Jyb3dzZXJOYW1lXS5jYXBhYmlsaXRpZXMubmFtZSkpIHtcbiAgICAgICAgICAgIGJvZHkubmFtZSA9IHRoaXMuc3VpdGVUaXRsZVxuICAgICAgICAgICAgaWYgKGJyb3dzZXJOYW1lKSB7XG4gICAgICAgICAgICAgICAgYm9keS5uYW1lID0gYCR7YnJvd3Nlck5hbWV9OiAke2JvZHkubmFtZX1gXG4gICAgICAgICAgICB9XG4gICAgICAgICAgICBpZiAoY2FsbGVkT25SZWxvYWQgfHwgdGhpcy50ZXN0Q250KSB7XG4gICAgICAgICAgICAgICAgbGV0IHRlc3RDbnQgPSArK3RoaXMudGVzdENudFxuICAgICAgICAgICAgICAgIGlmIChnbG9iYWwuYnJvd3Nlci5pc011bHRpcmVtb3RlKSB7XG4gICAgICAgICAgICAgICAgICAgIHRlc3RDbnQgPSBNYXRoLmNlaWwodGVzdENudCAvIGdsb2JhbC5icm93c2VyLmluc3RhbmNlcy5sZW5ndGgpXG4gICAgICAgICAgICAgICAgfVxuXG4gICAgICAgICAgICAgICAgYm9keS5uYW1lICs9IGAgKCR7dGVzdENudH0pYFxuICAgICAgICAgICAgfVxuICAgICAgICB9XG4gICAgICAgIGJvZHkuc3RhdHVzX2luZCA9IGZhaWx1cmVzID4gMCA/ICdmYWlsZWQnIDogJ3Bhc3NlZCdcbiAgICAgICAgcmV0dXJuIGJvZHlcbiAgICB9XG59Il19
```

Comment:
Let's remove the source map commands. The compiled script should automatically add this information.